lib/rules/rulesBase.ts

```diff
@@ -15,4 +17,34 @@ export class RuleBase {
   addMessage(diagnostic: OIDiagnostic): void {
     this.vhdlLinter.addMessage(diagnostic, (this.constructor as typeof RuleBase).ruleName);
   }
+}
+
+export function codeActionFromPrefixSuffix(token: OLexerToken, prefix: string, suffix: string, linter: VhdlLinter) {
+  // ignore implicit tokens
+  if (token.type === TokenType.implicit) {
+    return;
+  }
+  let newName = token.text;
+  if (prefix.trim().length !== 0 && newName.startsWith(prefix) === false) {
+    newName = `${prefix}${newName}`;
+  }
+  if (suffix.trim().length !== 0 && newName.endsWith(suffix) === false) {
+    newName = `${newName}${suffix}`;
+  }
```

Comment:
I like trimming the pre/suffix but should be done always? So `${prefix.trim()}${newName}`.
Also I am wondering should we make a settings normalization funktion or sth that does this automatically?
Doing this manually everywhere seems annoying?